content/courses/Guidelines_for_Safe_and_Secure_Ada_SPARK/rules/lkql/README.md

```diff
@@ -0,0 +1,27 @@
+The rules in this folder where specifically created for the
+Guidelines for Safe and Secure Ada/SPARK.
+
+Some of these rules are in the process of being added into the GNATcheck tool
+itself, while some of them are more related to this particular coding standard.
+The document is being updated (2024-10-08) to indicate which rules are built-in
+to GNATcheck, and which rules are specic to this document
+
+* function_out_parameters
+
+  * https://gitlab.adacore-it.com/eng/libadalang/langkit-query-language/-/issues/340
```

Comment:
The links to https://gitlab.adacore-it.com aren't accessible for everyone, so I'd rather leave them out.